actors/verifreg/tests/verifreg_actor_test.rs

```diff
@@ -178,133 +154,283 @@ mod verifiers {
 
     #[test]
     fn add_verifier_id_address() {
-        let (h, mut rt) = make_harness();
+        let (h, mut rt) = new_harness();
         h.add_verifier(&mut rt, &Address::new_id(201), &VERIFIER_ALLOWANCE).unwrap();
         h.check_state();
     }
 
     #[test]
     fn add_verifier_resolves_address() {
-        let (h, mut rt) = make_harness();
+        let (h, mut rt) = new_harness();
         let pubkey_addr = Address::new_secp256k1(&[0u8; 65]).unwrap();
         rt.id_addresses.insert(pubkey_addr, Address::new_id(201));
 
         h.add_verifier(&mut rt, &pubkey_addr, &VERIFIER_ALLOWANCE).unwrap();
         h.check_state();
     }
-}
 
-///// Test harness /////
+    #[test]
+    fn remove_requires_root() {
```

Comment:
nit: as ugly as the go MainTest/subtest-string names are when converting to rust it makes it easier to compare tests during review and make sure everything is written if we keep the names close to the same.  So here I would put "remove_verifier_fails_when_caller_is_not_the_root_key"

Reply:
Now that I'm seeing the wrapping `mod` approach the `remove_verifier` prefix could be dropped

Reply:
I won't rename these, but will adjust future names to be closer matches.